cob_voltage_control/package.xml

```diff
@@ -25,7 +25,8 @@
 
   <exec_depend condition="$ROS_PYTHON_VERSION == 2">python-matplotlib</exec_depend>
   <exec_depend condition="$ROS_PYTHON_VERSION == 3">python3-matplotlib</exec_depend>
-  <exec_depend>python-tk</exec_depend>
+  <exec_depend condition="$ROS_PYTHON_VERSION == 2">python-tk</exec_depend>
+  <exec_depend condition="$ROS_PYTHON_VERSION == 3">python3-tk</exec_depend>
```

Comment:
the key `python3-tk` does not exist in rosdistro
[`python-tk`] tries to distinguish between `python-tk` and `python3-tk` but uses `python-tk` with the joker notation `*` and `focal` is missing to be listed

Reply:
It does exist though:
https://github.com/ros/rosdistro/blob/a5f75c7667d603f643e87f72d5d5f8bbbf36dfb1/rosdep/python.yaml#L7327

Reply:
AFAIU  the use of `<distro>_python3` OSes is deprecated in rosdep because it requires users to override their OS definition with `--os` when invoking rosdep for it to take effect (which is not the case for debs).
The recommended way is to use the `python3-<...>` rosdep key for python3 dependencies et `python-<...>` for the python2 ones and use a condition if a package needs to use one or the other based on an external condition (like the ROS_PYTHON_VERSION)

Reply:
oh, I missed that

because our travis failed with:
```
install_target_dependencies
0.26sinstall_target_dependencies
'install_target_dependencies' returned with code '1' after 0 min 0 sec
The command ".industrial_ci/travis.sh" exited with 1.
Done. Your build exited with 1.
```
but thats due to the wrong package format in cob_bms_driver